components/compliance-service/ingest/ingestic/mappings/comp-sum-date.go

```diff
@@ -222,6 +230,14 @@ var ComplianceSumDate = Mapping{
                   },
                   "type": "object"
                 },
+                "waived": {
+                  "properties": {
+                    "total": {
```

Comment:
if we're only _adding_ es mappings, why do we need to do a migration?

Reply:
We are adding new aggregated fields in ElasticSearch, like:
```
            "waived": {
              "properties": {
                "total": {
                  "type": "integer"
                }
              },
```
which the elasticsearch queries (trend or back in time via date picker) for the compliance API calls will target. If those fields are missing for the old data, the queries will error out instead of saying 0 waived nodes 1 year ago.

Reply:
gotcha! thanks for the explanation!